SPECS/kernel/config

```diff
@@ -1534,7 +1534,6 @@ CONFIG_NET_SCHED=y
 #
 # Queueing/Scheduling
 #
-CONFIG_NET_SCH_CBQ=m
```

Comment:
Removed in commit [78bac52c290a7fc50607d4a91f828f1823a69824](https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/78bac52c290a7fc50607d4a91f828f1823a69824)